helpdesk_mgmt_sla_rule/readme/DESCRIPTION.rst

```diff
@@ -0,0 +1 @@
+This module improves the check sla by adding a sla.rule model.
```

Comment:
add a better description mentionning this module add a kind of rule engine for helpdesk SLA

add a usage file to explain how to use it and configure it